src/Build/BackEnd/Components/Communications/NodeProviderOutOfProcBase.cs

```diff
@@ -110,7 +110,7 @@ protected void ShutdownAllNodes(long hostHandshake, long clientHandshake, NodeCo
             // INodePacketFactory
             INodePacketFactory factory = new NodePacketFactory();
 
-            List<Process> nodeProcesses = GetPossibleRunningNodes();
+            List<Process> nodeProcesses = GetPossibleRunningNodes().Item2;
```

Comment:
Can I show you a [neat trick](https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-7#tuples)?

https://github.com/microsoft/msbuild/commit/44bdf900b6aac51c965aca60b6a6f932fca23d59